### PR TITLE
ascanrules: Update reference links (http -> https) Alert: 90019

### PR DESCRIPTION
**Pull request by 8linkz**
## Overview
I have only replaced http with https, the actual link has remained the same.
https://www.zaproxy.org/docs/alerts/90019-2/

## Related Issues
https://github.com/zaproxy/zaproxy/issues/8262

## Checklist
- [ ] Update help
- [x] Update changelog
- [ ] Run `./gradlew spotlessApply` for code formatting
- [ ] Write tests
- [ ] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

For more details, please refer to the [developer rules and guidelines](https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/).

### CODE REVIEW

**Comment by 8linkz** [comment]
The PR fixt also https://www.zaproxy.org/docs/alerts/90019-1/ as both share the same reference.

File: addOns/ascanrules/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Leave data empty instead of adding "N/A" for the scan rules:
   - Cross Site Scripting (Persistent) - Prime
   - Cross Site Scripting (Persistent) - Spider
+- Update reference for Server Side Code Injection - ASP Code Injection (Issue 8262).
```
Comment (thc202) [inline addOns/ascanrules/CHANGELOG.md:11]: Drop ` - ASP Code Injection`, it's changing the whole scan rule not just the ASP one.

Comment (8linkz) [reply addOns/ascanrules/CHANGELOG.md]: done

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**